subsys/logging/CMakeLists.txt

```diff
@@ -48,6 +47,16 @@ if(NOT CONFIG_LOG_MINIMAL)
     CONFIG_LOG_BACKEND_QEMU_X86_64
     log_backend_qemu_x86_64.c
   )
+
+  zephyr_sources_ifdef(
```

Comment:
That does not seems like the right approach here, @nordic-krch  can you please provide some feedback on how to support a different output back-end?

Reply:
@nashif IMHO back-end doesn't take care the output format, just pushing the data out with his own protocol(uart, usb, etc.). it's the place where in log_output.c logging subsystem decide the output format, so I replace the original one with sys-t format. I don't do the custom in the front-end, because I don't want to sys-t feature require more stack from the thread which use the front-end Marcos(LOG_INFO, LOG_ERR, etc).

@nordic-krch could you please help give your comments? Thanks.